skycoin/skywire
# Hypervisor dmsg tab.

Fixes #407

 Changes:	
- Added `/api/dmsg` endpoint to Hypervisor which tracks `dmsgctrl` connections.
- Visors now serves `dmsgctrl` streams.
- Fix segmentation fault due to incorrect address resolver init logic in `snet`.

How to test this PR:
- Start interactive testing environment with:
  - `skywire-services@feature/use-promutil`
  - `dmsg@develop`
- Navigate t0 `https://localhost:8000/api/dmsg`
- Attempts can be made to start/stop different visors and see changes to the output given by `https://localhost:8000/api/dmsg`. Note that some changes are not shown immediately.
